sdk/spring/azure-spring-integration-eventhubs/src/test/java/com/azure/spring/integration/eventhub/outbound/EventHubMessageHandlerTest.java

```diff
@@ -33,4 +57,72 @@ public void setUp() {
         this.handler = new DefaultMessageHandler(this.destination, this.sendOperation);
     }
 
+    private void initProperties() {
```

Comment:
move this into setup function will make it call before each test case

Reply:
Comments added for this method.